### ISSUE TITLE
Residential buildings do not update home count once plopped.

### ISSUE THREAD

**Issue by coldridj** [AJ3D/Ploppable-RICO#9]
Residential buildings maintain the home count that was configured when they were plopped, ignoring subsequent changes.

**Comment by originalfoo** [comment]
Do you have the WGs "realistic population and consumption" mod installed? If so, that overrides rooms/jobs defined in RICO settings unless the setting for a building says not to.

Also, did you remember to save local config after making changes?

After making changes, did you save and reload the game (might be related to #10)?

**Comment by coldridj** [comment]
I don't have that mod installed. Residential buildings plopped after modifying the home count have the correct value. It isn't being applied universally to all copies of that building.

**Comment by originalfoo** [comment]
I'd noticed similar issue but thought it might have been the WG mod - obviously not. 

I had some RICO [contemporary apartments](https://steamcommunity.com/sharedfiles/filedetails/?id=928884317) - some of them were giving 1 room, others had the 4 or 5 rooms that I'd set (depending on number of floors in the building).

Steps to reproduce (can't test at moment due to mod not compatible with GC patch) seem to be:

* Place RICO residential building
* Change room count & save RICO settings
* Already-placed buildings don't update
* New buildings get the new value?
* After save-load of the game, the older buildings still don't seem to update

**Comment by coldridj** [comment]
Yes those steps are accurate. I see there has been a new commit since I opened issue [#8](https://github.com/AJ3D/Ploppable-RICO/issues/8). I will see if I am now able to build the repository and attempt a fix myself.
